share/pvss/pvss_test.go

```diff
@@ -0,0 +1,310 @@
+package pvss
+
+import (
+	"testing"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/edwards"
+	"github.com/dedis/crypto/random"
+	"github.com/dedis/crypto/share"
+)
+
+func TestPVSS(test *testing.T) {
```

Comment:
Hmm, didn't see your comment until just now. I've now documented the usage scenario in the main comment of `pvss.go` and referred to the `pvss_test.go` for concrete examples. That okay?